show/kube.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python -u
+# -*- coding: utf-8 -*-
+    
+from show.main import AliasedGroup, cli, run_command
+import os
+from swsssdk import ConfigDBConnector
+
+KUBECTL_CMD = "kubectl --kubeconfig /etc/sonic/kube_admin.conf"
+REDIS_KUBE_TABLE = 'KUBERNETES_MASTER'
+REDIS_KUBE_KEY = 'SERVER'
+
+
+def hostname():
+    return os.uname()[1]
+
+def _get_db_data(table, key):
```

Comment:
>_get_db_data [](start = 4, length = 12)

_get_configdb_table_key

Reply:
Changed to _get_configdb_data. Table & key are implied through arg names.